diff -ruN a/mods/common/fluent/zh-CN/chrome.ftl b/mods/common/fluent/zh-CN/chrome.ftl
--- a/mods/common/fluent/zh-CN/chrome.ftl	1969-12-31 16:00:00
+++ b/mods/common/fluent/zh-CN/chrome.ftl	2026-07-30 21:51:09
@@ -0,0 +1,556 @@
+## assetbrowser.yaml
+label-assetbrowser-title = 素材浏览器
+label-assetbrowser-source-selector-desc = 选择素材来源
+dropdownbutton-assetbrowser-source-selector = 文件夹
+dropdownbutton-assetbrowser-asset-types-dropdown = 素材类型
+label-assetbrowser-filename-desc = 按名称过滤
+label-assetbrowser-sprite-scale = 缩放：
+label-assetbrowser-palette-desc = 调色板：
+label-assetbrowser-sprite-bg-error = 文件显示错误，详见 assetbrowser.log。
+
+## color-picker.yaml
+button-color-chooser-random = 随机
+button-color-chooser-store = 保存
+button-color-chooser-mixer-tab = 调色
+button-color-chooser-palette-tab = 色板
+label-preset-header = 预设颜色
+label-custom-header = 自定义颜色
+
+## confirmation-dialogs.yaml
+button-prompt-confirm = 确认
+button-prompt-other = 重启
+button-text-input-prompt-accept = 确定
+
+## connection.yaml
+label-connectionfailed-panel-password = 密码：
+button-connectionfailed-panel-abort = 中止
+label-connecting-panel-title = 正在连接
+button-connecting-panel-abort = 中止
+label-connection-switchmod-panel-title = 切换模组
+label-connection-switchmod-panel-desc = 该服务器正在运行不同的模组：
+label-connection-switchmod-panel-desc2 = 要切换模组并加入服务器吗？
+button-connection-switchmod-panel-switch = 切换
+button-connection-switchmod-panel-abort = 中止
+
+## credits.yaml, mainmenu.yaml
+label-openra = OpenRA
+label-credits-title = 制作名单
+
+## editor.yaml
+label-new-map-bg-title = 新建地图
+label-new-map-bg-tileset = 地形类型：
+label-new-map-bg-width = 宽度：
+label-new-map-bg-height = 高度：
+button-new-map-bg-create = 创建
+
+label-save-map-panel-heading = 保存地图
+label-save-map-panel-title = 标题：
+label-save-map-panel-author = 作者：
+label-save-map-panel-visibility = 可见性：
+dropdownbutton-save-map-panel-visibility-dropdown = 地图可见性
+label-save-map-panel-directory = 目录：
+label-save-map-panel-filename = 文件名：
+button-save-map-panel = 保存
+label-actor-edit-panel-id = ID
+button-container-ok = 确定
+label-tiles-bg-search = 搜索：
+label-bg-filter = 过滤：
+label-actors-bg-search = 搜索：
+label-actor-bg-owner = 所属方：
+label-area-selection = 区域选择
+label-area-info = 区域信息
+label-selected-area-diagonal = 对角线：
+label-selected-area-resources = 资源：
+label-selection-filters = 过滤器
+label-filter-terrain = 地形
+label-filter-resources = 资源
+label-filter-actors = 单位
+label-tool-bg-category = 工具：
+button-marker-tiles-clear-current = 清除当前
+button-marker-tiles-clear-all = 全部清除
+label-marker-layer-num-sides = 边数
+label-marker-alpha = 地块透明度
+label-marker-mirror-mode = 镜像模式
+label-marker-axis-angle = 轴角度
+
+button-map-editor-tab-container-select-tooltip = 选择
+button-map-editor-tab-container-tiles-tooltip = 地块
+button-map-editor-tab-container-overlays-tooltip = 覆盖物
+button-map-editor-tab-container-actors-tooltip = 单位
+button-map-editor-tab-container-tools-tooltip = 工具
+button-map-editor-tab-container-history-tooltip = 历史
+
+button-delete-area =
+    .label = 删除
+    .tooltip = 删除所选区域内的一切
+
+button-delete-actor =
+    .label = 删除
+    .tooltip = 删除选中的单位
+
+button-editor-world-root-options =
+    .label = 菜单
+    .tooltip = 菜单
+
+button-editor-world-root-copy =
+    .label = 复制
+    .tooltip = 复制所选区域
+
+button-editor-world-root-paste =
+    .label = 粘贴
+    .tooltip = 粘贴已保存的区域
+
+button-editor-world-root-undo =
+    .label = 撤销
+    .tooltip = 撤销上一步
+
+button-editor-world-root-redo =
+    .label = 重做
+    .tooltip = 重做上一步
+
+dropdownbutton-editor-world-root-overlay-button = 覆盖物
+button-select-categories-buttons-all = 全选
+button-select-categories-buttons-none = 全不选
+
+label-tool-marker-tiles = 标记地块
+
+## gamesave-browser.yaml
+label-gamesave-browser-panel-load-title = 载入游戏
+label-gamesave-browser-panel-save-title = 保存游戏
+label-gamesave-browser-panel-title = [新建存档]
+button-gamesave-browser-panel-delete-all = 全部删除
+button-gamesave-browser-panel-delete = 删除
+button-gamesave-browser-panel-rename = 重命名
+button-gamesave-browser-panel-load = 载入
+button-gamesave-browser-panel-save = 保存
+
+## ingame-chat.yaml, ingame-infochat.yaml
+button-chat-chrome-mode =
+    .label = 队伍
+    .tooltip = 切换聊天模式
+
+## ingame-debug-hpf.yaml
+dropdownbutton-hpf-overlay-locomotor = 选择移动方式
+dropdownbutton-hpf-overlay-check = 选择阻挡类型
+
+## ingame-debug.yaml
+label-debug-panel-title = 调试选项
+checkbox-debug-panel-instant-build = 瞬间建造
+checkbox-debug-panel-enable-tech = 全科技解锁
+checkbox-debug-panel-build-anywhere = 任意地点建造
+checkbox-debug-panel-unlimited-power = 无限电力
+checkbox-debug-panel-instant-charge = 瞬间充能
+checkbox-debug-panel-disable-visibility-checks = 禁用可见性检查
+button-debug-panel-give-cash = 获得 $20,000
+button-debug-panel-grow-resources = 资源生长
+button-debug-panel-give-exploration = 揭开战争迷雾
+button-debug-panel-reset-exploration = 重置战争迷雾
+label-debug-panel-visualizations-title = 可视化
+checkbox-debug-panel-show-unit-paths = 显示单位路径
+checkbox-debug-panel-show-customterrain-overlay = 显示自定义地形
+checkbox-debug-panel-show-actor-tags = 显示单位标签
+checkbox-debug-panel-show-combatoverlay = 显示战斗几何
+checkbox-debug-panel-show-geometry = 显示渲染几何
+checkbox-debug-panel-show-terrain-overlay = 显示地形几何
+checkbox-debug-panel-show-screenmap = 显示屏幕地图
+
+## ingame-infoobjectives.yaml
+label-mission-objectives = 任务：
+
+## ingame-infoscripterror.yaml
+label-script-error-panel-desc-a = 地图脚本发生致命错误
+label-script-error-panel-desc-b = 错误详情已保存至 logs 目录下的 lua.log。
+label-script-error-panel-desc-c = 请将该文件发送给地图作者以便修复。
+
+## ingame-infostats.yaml
+label-mission-objective = 任务：
+checkbox-stats-objective = 消灭所有敌人！
+label-stats-player = 玩家
+label-stats-faction = 阵营
+label-stats-score = 得分
+label-stats-actions = 操作
+
+## ingame-menu.yaml
+label-menu-buttons-title = 选项
+
+## lobby-kickdialogs.yaml
+label-kick-client-dialog-text-a = 你还可以同时施加临时封禁，
+label-kick-client-dialog-text-b = 使其在本局游戏内无法再次加入。
+checkbox-kick-client-dialog-prevent-rejoining = 临时封禁
+button-kick-client-dialog = 踢出
+label-kick-spectators-dialog-title = 踢出观战者
+button-kick-spectators-dialog-ok = 确定
+label-force-start-dialog-title = 开始游戏？
+label-force-start-dialog-text-a = 有玩家尚未准备就绪。
+label-force-start-dialog-text-b = 确定要强制开始游戏吗？
+label-kick-warning-a = 有客户端缺少所选地图，
+label-kick-warning-b = 他们将被移出服务器。
+button-force-start-dialog-start = 开始
+
+## lobby-mappreview.yaml
+label-map-incompatible-status-a = 此地图与此版本的 OpenRA
+label-map-incompatible-status-b = 不兼容
+label-map-validating-status = 正在校验……
+button-map-download-available-install = 安装地图
+button-map-preview-update = 更新地图
+button-map-update-download-available-install = 安装地图
+label-map-preview-searching = 正在搜索 OpenRA 资源中心……
+label-map-unavailable-a = 未在 OpenRA 资源中心
+label-map-unavailable-b = 找到此地图
+label-map-preview-error = 安装过程中发生错误
+label-map-update-available-a = 在你的电脑上发现了
+label-map-update-available-b = 该地图的新版本
+
+## lobby-music.yaml
+label-music-controls-volume = 音量：
+
+## lobby-music.yaml, lobby.yaml, mainmenu.yaml
+label-music-title = 音乐
+
+## lobby-music.yaml, musicplayer.yaml
+label-music-controls-length = 时长
+checkbox-music-controls-shuffle = 随机播放
+checkbox-music-controls-loop = 循环播放
+label-container-title = 曲目
+label-no-music-title = 未安装音乐
+label-no-music-desc-a = 可在「内容管理」菜单中
+label-no-music-desc-b = 安装游戏音乐。
+
+## lobby-options.yaml
+label-lobby-options-bin-title = 地图选项
+
+## lobby-players.yaml
+label-lobby-players-name = 名称
+label-lobby-players-color = 颜色
+label-lobby-players-faction = 阵营
+label-lobby-players-team = 队伍
+label-lobby-players-handicap = 让分
+label-lobby-players-spawn = 出生点
+label-lobby-players-ready = 就绪
+dropdownbutton-lobby-players-handicap-tooltip = 让分会降低该玩家部队的作战效能
+button-lobby-players-join = 加入此位置
+label-lobby-players-spectator = 观战者
+checkbox-lobby-players-new-spectator-toggle = 允许观战？
+button-lobby-players-spectate = 观战
+
+## lobby-servers.yaml
+image-lobby-servers-bin-password-protected-tooltip = 需要密码
+image-lobby-servers-bin-requires-authentication-tooltip = 需要 OpenRA 论坛账号
+dropdownbutton-lobby-servers-bin-filters = 筛选游戏
+
+## lobby-servers.yaml, multiplayer-browser.yaml
+label-container-server = 服务器
+label-container-players = 玩家
+label-container-location = 地区
+label-container-status = 状态
+label-notice-container-outdated-version = 你正在运行过期版本的 OpenRA，请前往 www.openra.net 下载最新版本
+label-notice-container-unknown-version = 你正在运行无法识别的 OpenRA 版本，请前往 www.openra.net 下载最新版本
+label-notice-container-playtest-available = 下一个 OpenRA 版本的预览版已开放测试，请前往 www.openra.net 下载 playtest
+
+## lobby.yaml
+dropdownbutton-server-lobby-slots = 位置管理
+button-server-lobby-reset-options = 恢复默认
+button-skirmish-tabs-players-tab = 玩家
+button-skirmish-tabs-options-tab = 选项
+button-multiplayer-tabs-players-tab = 玩家
+button-multiplayer-tabs-options-tab = 选项
+button-multiplayer-tabs-servers-tab = 服务器
+button-server-lobby-changemap = 更换地图
+
+button-lobbychat-chat-mode =
+    .label = 队伍
+    .tooltip = 切换聊天模式
+
+button-server-lobby-start-game = 开始游戏
+button-server-lobby-disconnect = 离开游戏
+
+## mainmenu-prompts.yaml
+label-mainmenu-introduction-prompt-title = 正在建立战场控制
+label-mainmenu-introduction-prompt-desc-a = 欢迎回来，指挥官！请使用以下选项初始化作战参数。
+label-mainmenu-introduction-prompt-desc-b = 更多选项可稍后在「设置」菜单中配置。
+label-mainmenu-system-info-prompt-title = 正在建立战场控制
+label-mainmenu-system-info-prompt-text-a = 我们希望收集一些系统信息，以帮助优化 OpenRA。
+label-mainmenu-system-info-prompt-text-b = 经你允许后，每次启动游戏时将发送以下匿名数据：
+checkbox-mainmenu-system-info-prompt-sysinfo = 发送系统信息
+
+## mainmenu-prompts.yaml, settings-display.yaml
+label-profile-section-header = 档案
+label-player-container = 玩家名称：
+label-playercolor-container-color = 偏好颜色：
+label-display-section-header = 显示
+label-battlefield-camera-dropdown = 战场镜头：
+label-ui-scale-dropdown = 界面缩放：
+checkbox-cursordouble-container = 放大鼠标指针
+
+## mainmenu-prompts.yaml, settings-input.yaml
+label-input-section-header = 输入
+label-mouse-control-container = 操作方案：
+label-mouse-control-desc-classic-selection = - 使用<左键>选择单位
+label-mouse-control-desc-classic-commands = - 使用<左键>指挥单位
+label-mouse-control-desc-classic-buildings = - 使用<左键>放置建筑
+label-mouse-control-desc-classic-support = - 使用<左键>施放支援技能
+label-mouse-control-desc-classic-zoom = - 使用<滚轮>缩放战场
+label-mouse-control-desc-classic-zoom-modifier = - 使用<MODIFIER + 滚轮>缩放战场
+label-mouse-control-desc-classic-scroll-right = - 使用<右键>平移战场
+label-mouse-control-desc-classic-scroll-middle = - 使用<中键>平移战场
+label-mouse-control-desc-classic-edgescroll = 或将光标移至屏幕边缘
+label-mouse-control-desc-modern-selection = - 使用<左键>选择单位
+label-mouse-control-desc-modern-commands = - 使用<右键>指挥单位
+label-mouse-control-desc-modern-buildings = - 使用<左键>放置建筑
+label-mouse-control-desc-modern-support = - 使用<左键>施放支援技能
+label-mouse-control-desc-modern-zoom = - 使用<滚轮>缩放战场
+label-mouse-control-desc-modern-zoom-modifier = - 使用<MODIFIER + 滚轮>缩放战场
+label-mouse-control-desc-modern-scroll-right = - 使用<右键>平移战场
+label-mouse-control-desc-modern-scroll-middle = - 使用<中键>平移战场
+label-mouse-control-desc-modern-edgescroll = 或将光标移至屏幕边缘
+checkbox-edgescroll-container = 屏幕边缘滚动
+
+## mainmenu.yaml
+label-singleplayer-title = 单人游戏
+button-extras-title = 更多
+button-main-menu-content = 内容管理
+button-singleplayer-menu-skirmish = 遭遇战
+button-singleplayer-menu-load = 载入
+button-extras-menu-replays = 回放
+label-map-editor-title = 地图编辑器
+button-extras-menu-assetbrowser = 素材浏览器
+button-map-editor-new-map = 新建地图
+button-map-editor-load-map = 载入地图
+dropdownbutton-news-bg-button = 战场新闻
+label-update-notice-a = 你正在运行过期版本的 OpenRA。
+label-update-notice-b = 请前往 www.openra.net 下载最新版本
+
+## mainmenu.yaml, missionbrowser.yaml
+label-missions-title = 任务
+
+## mainmenu.yaml, multiplayer-browser.yaml
+label-multiplayer-title = 多人游戏
+
+## mainmenu.yaml, settings.yaml
+button-settings-title = 设置
+
+## map-chooser.yaml
+label-mapchooser-panel-title = 选择地图
+button-mapchooser-panel-system-maps-tab = 官方地图
+button-mapchooser-panel-remote-maps-tab = 服务器地图
+button-mapchooser-panel-user-maps-tab = 自定义地图
+label-filter-order-controls-desc = 筛选：
+label-filter-order-controls-desc-joiner = 于
+label-filter-order-controls-orderby = 排序：
+button-mapchooser-panel-randommap = 随机地图
+button-mapchooser-panel-delete-map = 删除地图
+button-mapchooser-panel-delete-all-maps = 删除全部地图
+button-mapchooser-panel-ok = 确定
+
+## missionbrowser.yaml
+button-missionbrowser-panel-start-briefing-video = 观看简报
+button-missionbrowser-panel-stop-briefing-video = 停止简报
+button-missionbrowser-panel-start-info-video = 观看情报视频
+button-missionbrowser-panel-stop-info-video = 停止情报视频
+button-missionbrowser-panel-play = 开始
+button-missionbrowser-panel-mission-info = 任务情报
+button-missionbrowser-panel-mission-options = 选项
+dropdown-missionbrowser-difficulty =
+    .label = 难度
+    .description = 任务难度
+
+dropdown-missionbrowser-gamespeed = 速度：
+label-missionbrowser-normal-difficulty = 普通
+
+## multiplayer-browser.yaml
+image-multiplayer-panel-password-protected-tooltip = 需要密码
+image-multiplayer-panel-requires-authentication-tooltip = 需要 OpenRA 论坛账号
+button-selected-server-join = 加入
+dropdownbutton-multiplayer-panel-filters = 筛选游戏
+button-multiplayer-panel-directconnect = IP 直连
+button-multiplayer-panel-create = 创建
+
+## multiplayer-browserpanels.yaml
+checkbox-multiplayer-filter-panel-waiting-for-players = 等待中
+checkbox-multiplayer-filter-panel-empty = 空房间
+checkbox-multiplayer-filter-panel-password-protected = 有密码
+checkbox-multiplayer-filter-panel-already-started = 已开始
+checkbox-multiplayer-filter-panel-incompatible-version = 版本不兼容
+
+## multiplayer-createserver.yaml
+label-multiplayer-createserver-panel-title = 创建服务器
+label-multiplayer-createserver-panel-server-name = 服务器名称：
+label-multiplayer-createserver-panel-password = 密码：
+label-multiplayer-createserver-panel-after-password = （可选）
+label-multiplayer-createserver-panel-listen-port = 端口：
+checkbox-multiplayer-createserver-panel-advertise = 在线广播
+label-notices-lan-advertising = - 游戏将仅在局域网内广播。
+label-notices-lan-firewall = - 你需要手动配置防火墙以允许连接。
+label-notices-lan-portforward-a = - 若在路由器上手动配置端口转发，
+label-notices-lan-portforward-b = 玩家可通过互联网使用 IP 直连加入。
+label-notices-no-upnp-advertising = - 游戏将在局域网和互联网上广播。
+label-notices-no-upnp-firewall = - 你需要手动配置防火墙以允许连接。
+label-notices-no-upnp-portforward-a = - 你需要手动配置路由器，
+label-notices-no-upnp-portforward-b = 允许并转发到本机 IP 与端口。
+label-notices-no-upnp-settings-a = - 你可以在设置菜单的「高级」选项卡中
+label-notices-no-upnp-settings-b = 启用 UPnP/NAT-PMP（若路由器支持）。
+label-notices-upnp-advertising = - 游戏将在局域网和互联网上广播。
+label-notices-upnp-firewall = - 你需要手动配置防火墙以允许连接。
+label-notices-upnp-portforward-a = - 游戏将自动配置端口转发。
+label-notices-upnp-settings-a = - 你可以在设置菜单中禁用 UPnP/NAT-PMP。
+button-multiplayer-createserver-panel-change-map = 更换地图
+button-multiplayer-createserver-panel-create = 创建
+
+## multiplayer-directconnect.yaml
+label-directconnect-panel-title = 连接到服务器
+label-directconnect-panel-address = 服务器地址：
+label-directconnect-panel-port = ：
+button-directconnect-panel-join = 加入
+
+## playerprofile.yaml
+button-profile-header-logout = 退出登录
+label-generate-keys-desc-a = 连接论坛账号以向其他玩家
+label-generate-keys-desc-b = 表明身份、加入私密服务器
+label-generate-keys-desc-c = 并展示徽章。
+button-generate-keys-key = 连接 OpenRA 论坛账号
+label-generating-keys-desc-a = 正在生成认证密钥对。
+label-generating-keys-desc-b = 这将需要几秒钟……
+label-register-fingerprint-desc-a = 认证密钥已复制到你的剪贴板。
+label-register-fingerprint-desc-b = 请将其添加到 OpenRA 论坛的
+label-register-fingerprint-desc-c = 用户控制面板，然后点「继续」。
+label-checking-fingerprint-desc-a = 正在从 OpenRA 论坛
+label-checking-fingerprint-desc-b = 查询账号信息……
+label-fingerprint-not-found-desc-a = 你的认证密钥尚未连接
+label-fingerprint-not-found-desc-b = 到 OpenRA 论坛账号。
+label-connection-error-desc-a = 无法连接到 OpenRA 论坛。
+label-connection-error-desc-b = 请检查你的网络连接。
+
+## replaybrowser.yaml
+label-replaybrowser-panel-title = 回放查看器
+label-filters-title = 筛选
+label-filters-flt-gametype-desc = 类型：
+dropdownbutton-filters-any = 任意
+label-filters-flt-date-desc = 日期：
+label-filters-flt-duration-desc = 时长：
+label-filters-flt-mapname-desc = 地图：
+label-filters-flt-player-desc = 玩家：
+dropdownbutton-filters-flt-player = 任何人
+label-filters-flt-outcome-desc = 结果：
+label-filters-flt-faction-desc = 阵营：
+button-filters-flt-reset = 重置筛选
+label-management-manage-title = 管理
+button-management-mng-rensel = 重命名
+button-management-mng-delsel = 删除
+button-management-mng-delall = 全部删除
+label-replay-list-container-replaybrowser-title = 选择回放
+button-replaybrowser-panel-watch = 观看
+
+## settings-advanced.yaml
+label-network-section-header = 高级
+checkbox-nat-discovery-container = 启用 UPnP/NAT-PMP 发现
+checkbox-fetch-news-container = 获取社区新闻
+checkbox-perfgraph-container = 显示性能图表
+checkbox-check-version-container = 检查更新
+checkbox-perftext-container = 显示性能文本
+checkbox-sendsysinfo-container = 发送系统信息
+label-sendsysinfo-checkbox-container-desc = 你的操作系统、OpenGL 与 .NET 运行时版本及语言设置将随匿名 ID 一并发送，用于排定未来开发优先级。
+label-debug-section-header = 开发者
+label-debug-hidden-container-a = 更多开发者选项可通过
+label-debug-hidden-container-b = Debug.DisplayDeveloperSettings 设置或启动参数启用
+checkbox-botdebug-container = 显示 Bot 调试信息
+checkbox-checkbotsync-container = 检查 BotModule 代码同步
+checkbox-luadebug-container = 显示地图调试信息
+checkbox-checkunsynced-container = 检查非同步代码
+checkbox-replay-commands-container = 回放中启用调试命令
+checkbox-perflogging-container = 启用 Tick 性能日志
+
+## settings-audio.yaml
+label-audio-section-header = 音频
+label-no-audio-device-container = 音频控制需要可用的声音设备
+checkbox-cash-ticks-container = 资金音效
+checkbox-mute-sound-container = 静音
+label-sound-volume-container = 音效音量：
+
+checkbox-mute-background-music-container =
+    .label = 静音菜单音乐
+    .tooltip = 没有特定曲目播放时静音背景音乐
+
+label-music-title-volume-container = 音乐音量：
+label-audio-device-container = 音频设备：
+label-video-volume-container = 视频音量：
+label-restart-required-container-audio-desc = 设备变更将在重启游戏后生效
+
+## settings-display.yaml
+label-target-lines-dropdown-container = 目标指示线：
+label-status-bar-dropdown-container-bars = 状态栏：
+
+checkbox-player-stance-colors-container =
+    .label = 玩家关系颜色
+    .tooltip = 根据关系（自己、敌方、盟友、中立）改变玩家颜色
+
+checkbox-ui-feedback-container =
+    .label = 显示界面反馈通知
+    .tooltip = 为界面事件显示短暂的文字通知
+
+checkbox-transients-container =
+    .label = 显示游戏事件通知
+    .tooltip = 为游戏事件显示短暂的文字通知
+
+checkbox-pause-shellmap-container = 暂停菜单背景
+checkbox-hide-replay-chat-container = 回放中隐藏聊天
+label-video-section-header = 视频
+label-video-mode-dropdown-container = 视频模式：
+dropdownbutton-video-mode-dropdown-container = 窗口化
+label-language-dropdown-container = 语言（重启后生效）：
+label-window-resolution-container-size = 窗口尺寸：
+label-window-resolution-container-x = x
+label-display-selection-container = 选择显示器：
+dropdownbutton-display-selection-container-dropdown = 标准
+checkbox-vsync-container = 启用垂直同步
+checkbox-frame-limit-gamespeed-container = 帧率限制为游戏逻辑速率
+label-gl-profile-dropdown-container = OpenGL 配置：
+label-restart-required-container-video-desc = 显示与 OpenGL 变更需要重启
+
+## settings-hotkeys.yaml
+hotkey-group-game-commands = 游戏命令
+hotkey-group-viewport-commands = 视角命令
+hotkey-group-observer-replay-commands = 观战 / 回放命令
+hotkey-group-unit-commands = 单位命令
+hotkey-group-unit-stance-commands = 单位状态命令
+hotkey-group-production-commands = 生产命令
+hotkey-group-support-power-commands = 支援技能命令
+hotkey-group-music-commands = 音乐命令
+hotkey-group-chat-commands = 聊天命令
+hotkey-group-control-groups = 编组
+hotkey-group-editor-commands = 编辑器命令
+label-hotkeys-panel-filter-input = 按名称筛选：
+label-hotkeys-panel-context-dropdown = 上下文：
+label-hotkey-empty-list-message = 没有符合筛选条件的热键。
+label-notices-readonly-notice = 此热键不可修改
+button-hotkey-remap-dialog-override = 覆盖
+
+button-hotkey-remap-dialog-clear =
+    .label = 清除
+    .tooltip = 解除该热键绑定
+
+button-hotkey-remap-dialog-reset =
+    .label = 重置
+    .tooltip = 恢复默认
+
+## settings-input.yaml
+label-zoom-modifier-container = 缩放修饰键：
+checkbox-alternate-scroll-container = 备用鼠标平移
+checkbox-lockmouse-container = 锁定鼠标于窗口内
+label-mouse-scroll-type-container = 平移方式：
+label-scrollspeed-slider-container-scroll-speed = 平移速度：
+label-zoomspeed-slider-container-zoom-speed = 缩放速度：
+label-ui-scrollspeed-slider-container-scroll-speed = 界面滚动速度：
+
+## settings.yaml
+button-settings-panel-reset = 重置
+
+## tooltips.yaml
+label-latency-tooltip-prefix = 延迟：
+label-anonymous-player-tooltip-name = 匿名玩家
+label-bot-player-tooltip-name = 电脑
+label-bot-managed-by-tooltip = 电脑由 { $name } 管理
+label-game-admin = 游戏管理员
+
+## gamesave-loading.yaml
+label-gamesave-loading-screen-title = 正在载入存档
+label-gamesave-loading-screen-desc = 按 Esc 取消载入并返回主菜单
diff -ruN a/mods/common/fluent/zh-CN/common.ftl b/mods/common/fluent/zh-CN/common.ftl
--- a/mods/common/fluent/zh-CN/common.ftl	1969-12-31 16:00:00
+++ b/mods/common/fluent/zh-CN/common.ftl	2026-07-30 21:51:09
@@ -0,0 +1,857 @@
+## Buttons
+button-cancel = 取消
+button-retry = 重试
+button-back = 返回
+button-continue = 继续
+button-quit = 退出
+
+## Server Orders
+notification-custom-rules = 此地图包含自定义规则，游戏体验可能有所变化。
+notification-map-bots-disabled = 此地图已禁用电脑玩家。
+notification-two-humans-required = 此服务器需要至少两名人类玩家才能开始比赛。
+notification-unknown-server-command = 未知的服务器命令：{ $command }。
+notification-admin-start-game = 只有主机才能开始游戏。
+notification-no-start-until-required-slots-full = 必需的位置未满，无法开始游戏。
+notification-no-start-without-players = 没有玩家，无法开始游戏。
+notification-insufficient-enabled-spawn-points = 启用的出生点不足，无法开始游戏。
+notification-malformed-command = { $command } 命令格式错误。
+notification-state-unchanged-ready = 标记为就绪后无法更改状态。
+notification-invalid-faction-selected = 无效的阵营选择：{ $faction }。
+notification-state-unchanged-game-started = 游戏开始后无法更改状态（{ $command }）。
+notification-requires-host = 只有主机才能执行此操作。
+notification-invalid-bot-slot = 无法向已有玩家的位置添加电脑。
+notification-invalid-bot-type = 无效的电脑类型。
+notification-admin-change-map = 只有主机才能更换地图。
+notification-player-disconnected = { $player } 已断开连接。
+notification-team-player-disconnected = { $player }（队伍 { $team }）已断开连接。
+notification-observer-disconnected = { $player }（观战者）已断开连接。
+notification-unknown-map = 服务器上未找到该地图。
+notification-searching-map = 正在资源中心搜索地图……
+notification-admin-change-configuration = 只有主机才能更改配置。
+notification-changed-map = { $player } 将地图更换为 { $map }。
+notification-option-changed = { $player } 将 { $name } 更改为 { $value }。
+notification-you-were-kicked = 你已被踢出服务器。
+notification-admin-kicked = { $admin } 将 { $player } 踢出了服务器。
+notification-kicked = { $player } 已被踢出服务器。
+notification-temp-ban = { $admin } 对 { $player } 实施了临时封禁。
+notification-admin-transfer-admin = 只有管理员才能转让管理员权限。
+notification-admin-move-spectators = 只有主机才能将玩家移至观战。
+notification-empty-slot = 该位置无人。
+notification-move-spectators = { $admin } 将 { $player } 移至观战。
+notification-nick-changed = { $player } 现已更名为 { $name }。
+notification-player-dropped = 一名玩家因超时被移除。
+notification-connection-problems = { $player } 遇到连接问题。
+notification-timeout-dropped = { $player } 因超时已被移除。
+notification-timeout-dropped-in =
+    { $timeout ->
+       *[other] { $player } 将在 { $timeout } 秒后被移除。
+    }
+notification-error-game-started = 游戏已经开始。
+notification-requires-password = 服务器需要密码。
+notification-incorrect-password = 密码错误。
+notification-incompatible-mod = 服务器正在运行不兼容的模组。
+notification-incompatible-version = 服务器正在运行不兼容的版本。
+notification-incompatible-protocol = 服务器正在运行不兼容的协议。
+notification-you-were-banned = 你已被服务器封禁。
+notification-you-were-temp-banned = 你已被服务器临时封禁。
+notification-game-full = 游戏已满员。
+notification-new-admin = { $player } 现在是管理员。
+notification-option-locked = { $option } 无法更改。
+notification-invalid-configuration-command = 无效的配置命令。
+notification-admin-option = 只有主机才能设置该选项。
+notification-error-number-teams = 无法解析队伍数量：{ $raw }。
+notification-admin-kick = 只有主机才能踢出玩家。
+notification-kick-self = 主机不能踢出自己。
+notification-kick-none = 该位置无人。
+notification-no-kick-game-started = 游戏开始后只能踢出观战者和已战败的玩家。
+notification-admin-clear-spawn = 只有管理员才能清除出生点。
+notification-spawn-occupied = 你不能与其他玩家占用同一出生点。
+notification-spawn-locked = 该出生点已锁定给其他玩家位置。
+notification-admin-lobby-info = 只有主机才能设置大厅信息。
+notification-invalid-lobby-info = 发送了无效的大厅信息。
+notification-player-color-terrain = 颜色已调整以减少与地形的相似度。
+notification-player-color-player = 颜色已调整以减少与其他玩家的相似度。
+notification-invalid-player-color = 无法确定有效的玩家颜色，已随机选择。
+notification-invalid-error-code = 解析错误消息失败。
+notification-master-server-connected = 主服务器通信已建立。
+notification-master-server-error = 主服务器通信失败。
+notification-game-offline = 游戏未在线广播。
+notification-no-port-forward = 服务器端口无法从互联网访问。
+notification-blacklisted-server-name = 服务器名称包含违禁词。
+notification-requires-authentication = 服务器要求玩家拥有 OpenRA 论坛账号。
+notification-no-permission-to-join = 你没有加入此服务器的权限。
+notification-slot-closed = 你的位置已被主机关闭。
+
+## LobbySettingsNotification
+notification-lobby-option = { $name }：{ $value }。
+
+## ServerOrders, UnitOrders
+notification-joined = { $player } 加入了游戏。
+notification-lobby-disconnected = { $player } 已离开。
+
+## UnitOrders
+notification-game-has-started = 游戏已开始。
+notification-game-saved = 游戏已保存。
+notification-game-paused = 游戏已被 { $player } 暂停。
+notification-game-unpaused = 游戏已被 { $player } 恢复。
+
+## Server
+notification-game-started = 游戏开始。
+
+## PlayerMessageTracker
+notification-chat-temp-disabled =
+    { $remaining ->
+       *[other] 聊天已禁用，请在 { $remaining } 秒后重试。
+    }
+
+## VoteKickTracker
+notification-unable-to-start-a-vote = 无法发起投票。
+notification-insufficient-votes-to-kick = 踢出玩家 { $kickee } 的票数不足。
+notification-kick-already-voted = 你已经投过票了。
+notification-vote-kick-started = 玩家 { $kicker } 发起了踢出 { $kickee } 的投票。
+notification-vote-kick-in-progress = { $percentage }% 的玩家已投票踢出 { $kickee }。
+notification-vote-kick-ended = 踢出玩家 { $kickee } 的投票未通过。
+
+## ActorEditLogic
+label-duplicate-actor-id = 单位 ID 重复
+label-actor-id = 输入单位 ID
+label-actor-owner = 所属方
+
+## ActorSelectorLogic
+label-actor-type = 类型：{ $actorType }
+
+## CommonSelectorLogic
+options-common-selector =
+    .search-results = 搜索结果
+    .all = 全部
+    .multiple = 多个
+    .none = 无
+
+## SaveMapLogic
+label-unpacked-map = 未打包
+
+dialog-save-map-failed =
+    .title = 地图保存失败
+    .prompt = 详见 debug.log。
+    .confirm = 确定
+
+dialog-overwrite-map-failed =
+    .title = 警告
+    .prompt = 保存将覆盖
+    已存在的地图。
+    .confirm = 保存
+
+dialog-overwrite-map-outside-edit =
+    .title = 警告
+    .prompt = 该地图已在编辑器之外被修改，
+    保存可能会覆盖已有进度。
+    .confirm = 保存
+
+notification-save-current-map = 已保存当前地图。
+
+## GameInfoLogic
+menu-game-info =
+    .objectives = 目标
+    .briefing = 简报
+    .options = 选项
+    .debug = 调试
+    .chat = 聊天
+
+## GameInfoObjectivesLogic, GameInfoStatsLogic
+label-mission-in-progress = 进行中
+label-mission-accomplished = 已完成
+label-mission-failed = 已失败
+
+## GameInfoStatsLogic
+label-client-state-disconnected = 离线
+label-mute-player = 屏蔽此玩家
+label-unmute-player = 取消屏蔽此玩家
+button-kick-player = 踢出此玩家
+button-vote-kick-player = 投票踢出此玩家
+
+dialog-kick =
+    .title = 踢出 { $player }？
+    .prompt = 该玩家将无法重新加入游戏。
+    .confirm = 踢出
+
+dialog-vote-kick =
+    .title = 投票踢出 { $player }？
+    .prompt = 该玩家将无法重新加入游戏。
+    .prompt-break-bots =
+    { $bots ->
+        [one] 踢出游戏管理员将同时踢出 1 个电脑。
+       *[other] 踢出游戏管理员将同时踢出 { $bots } 个电脑。
+    }
+    .vote-start = 发起投票
+    .vote-for = 赞成
+    .vote-against = 反对
+    .vote-cancel = 弃权
+
+notification-vote-kick-disabled = 此服务器已禁用投票踢人。
+
+## GameTimerLogic
+label-paused = 已暂停
+label-max-speed = 最高速度
+label-replay-speed = { $percentage }% 速度
+label-replay-complete = 已完成 { $percentage }%
+
+## LobbyLogic, InGameChatLogic
+label-chat-disabled = 聊天已禁用
+label-chat-availability =
+    { $seconds ->
+       *[other] 聊天将在 { $seconds } 秒后可用……
+    }
+
+## LobbyLogic, ServerListLogic
+label-bot-player = AI 玩家
+
+## IngameMenuLogic
+menu-ingame =
+    .leave = 离开
+    .abort = 放弃任务
+    .restart = 重新开始
+    .surrender = 投降
+    .load-game = 载入游戏
+    .save-game = 保存游戏
+    .music = 音乐
+    .settings = 设置
+    .return-to-map = 返回地图
+    .resume = 继续
+    .save-map = 保存地图
+    .exit-map = 退出地图编辑器
+
+dialog-leave-mission =
+    .title = 离开任务
+    .prompt = 离开本局并返回菜单？
+    .confirm = 离开
+    .cancel = 留下
+
+dialog-restart-mission =
+    .title = 重新开始
+    .prompt = 确定要重新开始吗？
+    .confirm = 重新开始
+    .cancel = 留下
+
+dialog-surrender =
+    .title = 投降
+    .prompt = 确定要投降吗？
+    .confirm = 投降
+    .cancel = 留下
+
+dialog-error-max-player =
+    .title = 错误：玩家数量超限
+    .prompt = 定义的玩家过多（{ $players }/{ $max }）。
+    .confirm = 返回
+
+dialog-exit-map-editor =
+    .title = 退出地图编辑器
+    .prompt-unsaved = 退出并丢失所有未保存的更改？
+    .prompt-deleted = 该地图可能已在编辑器之外被删除
+    .confirm-anyway = 仍然退出
+    .confirm = 退出
+
+dialog-play-map-warning =
+    .title = 警告
+    .prompt = 该地图可能已被删除，或存在
+    导致无法加载的错误。
+    .cancel = 确定
+
+dialog-exit-to-map-editor =
+    .title = 离开任务
+    .prompt = 离开本局并返回编辑器？
+    .confirm = 返回编辑器
+    .cancel = 留下
+
+## IngamePowerBarLogic
+## IngamePowerCounterLogic
+label-power-usage = 电力使用：{ $usage }/{ $capacity }
+label-infinite-power = 无限
+
+## IngameSiloBarLogic
+## IngameCashCounterLogic
+label-silo-usage = 储存容量：{ $usage }/{ $capacity }
+
+## ObserverShroudSelectorLogic
+options-shroud-selector =
+    .all-players = 所有玩家
+    .disable-shroud = 关闭迷雾
+    .other = 其他
+
+## ObserverStatsLogic
+options-observer-stats =
+    .none = 信息：无
+    .basic = 基本
+    .economy = 经济
+    .production = 生产
+    .support-powers = 支援技能
+    .combat = 战斗
+    .army = 军队
+    .earnings-graph = 收入（图表）
+    .army-graph = 军队（图表）
+
+## WorldTooltipLogic
+label-unrevealed-terrain = 未探索区域
+
+## DownloadPackageLogic
+label-downloading = 正在下载 { $title }
+label-fetching-mirror-list = 正在获取镜像列表……
+label-downloading-from = 正在从 { $host } 下载 { $received } { $suffix }
+label-downloading-from-progress = 正在从 { $host } 下载 { $received } / { $total } { $suffix }（{ $progress }%）
+label-unknown-host = 未知主机
+label-download-failed = 下载失败
+label-verifying-archive = 正在校验压缩包……
+label-archive-validation-failed = 压缩包校验失败
+label-extracting-archive = 正在解压……
+label-extracting-archive-entry = 正在解压 { $entry }
+label-archive-extraction-failed = 压缩包解压失败
+label-mirror-selection-failed = 在线镜像不可用，请从原版光盘安装。
+
+## InstallFromSourceLogic
+label-detecting-sources = 正在检测驱动器
+label-checking-sources = 正在检查来源
+label-searching-source-for = 正在搜索 { $title }
+label-content-package-installation = 选择要安装的内容包：
+label-game-sources = 游戏来源
+label-digital-installs = 数字版安装
+label-game-content-not-found = 未找到游戏内容
+label-alternative-content-sources = 请插入或安装以下内容来源之一：
+label-installing-content = 正在安装内容
+label-copying-filename = 正在复制 { $filename }
+label-copying-filename-progress = 正在复制 { $filename }（{ $progress }%）
+label-installation-failed = 安装失败
+label-check-install-log = 详见 logs 目录下的 install.log。
+label-extracting-filename = 正在解压 { $filename }
+label-extracting-filename-progress = 正在解压 { $filename }（{ $progress }%）
+
+## ModContentLogic
+button-manual-install = 手动安装
+
+## KickClientLogic
+dialog-kick-client =
+    .prompt = 踢出 { $player }？
+
+## KickSpectatorsLogic
+dialog-kick-spectators =
+    .prompt =
+    { $count ->
+        [one] 确定要踢出 1 名观战者吗？
+       *[other] 确定要踢出 { $count } 名观战者吗？
+    }
+
+## LobbyLogic
+options-slot-admin =
+    .add-bots = 添加
+    .remove-bots = 移除
+    .configure-bots = 配置电脑
+    .teams-count = { $count } 支队伍
+    .humans-vs-bots = 玩家对电脑
+    .free-for-all = 各自为战
+    .configure-teams = 配置队伍
+
+## LobbyLogic, InGameChatLogic
+button-general-chat = 全体
+button-team-chat = 队伍
+
+## LobbyOptionsLogic, MissionBrowserLogic
+label-not-available = 不可用
+
+## LobbyUtils
+options-lobby-slot =
+    .slot = 位置
+    .open = 开放
+    .closed = 关闭
+    .bots = 电脑
+    .bots-disabled = 已禁用电脑
+
+## MapPreviewLogic
+label-connecting = 正在连接……
+label-downloading-map = 正在下载 { $size } kB
+label-downloading-map-progress = 正在下载 { $size } kB（{ $progress }%）
+button-retry-install = 重试安装
+button-retry-search = 重试搜索
+## also MapChooserLogic
+label-created-by = 作者：{ $author }
+
+## SpawnSelectorTooltipLogic
+label-disabled-spawn = 已禁用的出生点
+label-available-spawn = 可用的出生点
+
+## DisplaySettingsLogic
+options-camera =
+    .close = 近
+    .medium = 中
+    .far = 远
+    .furthest = 最远
+
+options-display-mode =
+    .windowed = 窗口化
+    .legacy-fullscreen = 全屏（旧式）
+    .fullscreen = 全屏
+
+label-video-display-index = 显示器 { $number }
+
+options-status-bars =
+    .standard = 标准
+    .show-on-damage = 受伤时显示
+    .always-show = 始终显示
+
+options-target-lines =
+    .automatic = 自动
+    .manual = 手动
+    .disabled = 禁用
+
+checkbox-frame-limiter = 启用帧率限制（{ $fps } FPS）
+
+## HotkeysSettingsLogic
+label-original-notice = 默认为「{ $key }」
+label-duplicate-notice = 该键已在 { $context } 上下文中用于「{ $key }」
+hotkey-context-any = 任意
+
+## InputSettingsLogic
+options-mouse-scroll-type =
+    .disabled = 禁用
+    .standard = 标准
+    .inverted = 反转
+    .joystick = 摇杆
+
+## InputSettingsLogic, IntroductionPromptLogic
+options-control-scheme =
+    .classic = 经典
+    .modern = 现代
+
+## SettingsLogic
+dialog-settings-save =
+    .title = 需要重启
+    .prompt = 部分更改将在
+    重启游戏后生效。
+    .cancel = 继续
+
+dialog-settings-restart =
+    .title = 立即重启？
+    .prompt = 部分更改将在重启游戏后
+    生效。现在重启吗？
+    .confirm = 立即重启
+    .cancel = 稍后重启
+
+dialog-settings-reset =
+    .title = 重置{ $panel }
+    .prompt = 确定要重置
+    此面板中的所有设置吗？
+    .confirm = 重置
+    .cancel = 取消
+
+## AssetBrowserLogic
+label-all-packages = 所有包
+label-length-in-seconds = { $length } 秒
+
+## ConnectionLogic
+label-connecting-to-endpoint = 正在连接 { $endpoint }……
+label-could-not-connect-to-target = 无法连接到 { $target }
+label-unknown-error = 未知错误
+label-password-required = 需要密码
+label-connection-failed = 连接失败
+notification-mod-switch-failed = 模组切换失败。
+
+## GameSaveBrowserLogic
+dialog-rename-save =
+    .title = 重命名存档
+    .prompt = 输入新文件名：
+    .confirm = 重命名
+
+dialog-delete-save =
+    .title = 删除所选存档？
+    .prompt = 删除「{ $save }」。
+    .confirm = 删除
+
+dialog-delete-all-saves =
+    .title = 删除全部存档？
+    .prompt =
+    { $count ->
+       *[other] 删除 { $count } 个存档。
+    }
+    .confirm = 全部删除
+
+notification-save-deletion-failed = 删除存档「{ $savePath }」失败，详见日志。
+
+dialog-overwrite-save =
+    .title = 覆盖存档？
+    .prompt = 覆盖 { $file }？
+    .confirm = 覆盖
+
+## MainMenuLogic
+label-loading-news = 正在加载新闻
+label-news-retrieval-failed = 获取新闻失败：{ $message }
+label-news-parsing-failed = 解析新闻失败：{ $message }
+label-author-datetime = { $author } 发布于 { $datetime }
+
+## MapChooserLogic
+label-all-maps = 所有地图
+label-no-matches = 无匹配
+label-player-count =
+    { $players ->
+       *[other] { $players } 名玩家
+    }
+label-map-size-huge = （巨大）
+label-map-size-large = （大）
+label-map-size-medium = （中）
+label-map-size-small = （小）
+label-map-searching-count =
+    { $count ->
+       *[other] 正在 OpenRA 资源中心搜索 { $count } 张地图……
+    }
+label-map-unavailable-count =
+    { $count ->
+       *[other] { $count } 张地图未在 OpenRA 资源中心找到
+    }
+
+notification-map-deletion-failed = 删除地图「{ $map }」失败，详见 debug.log。
+
+dialog-delete-map =
+    .title = 删除地图
+    .prompt = 删除地图「{ $title }」？
+    .confirm = 删除
+
+dialog-delete-all-maps =
+    .title = 删除地图
+    .prompt = 删除本页所有地图？
+    .confirm = 删除
+
+options-order-maps =
+    .player-count = 玩家
+    .title = 标题
+    .date = 日期
+    .size = 大小
+
+## MissionBrowserLogic
+dialog-no-video =
+    .title = 未安装视频
+    .prompt = 可在模组选择器的
+    「内容管理」菜单中安装游戏视频。
+    .cancel = 返回
+
+dialog-cant-play-video =
+    .title = 无法播放视频
+    .prompt = 视频播放时出现问题。
+    .cancel = 返回
+
+## MusicPlayerLogic
+label-sound-muted = 音频已在设置中静音。
+label-no-song-playing = 没有正在播放的曲目
+
+## MuteHotkeyLogic
+label-audio-muted = 音频已静音。
+label-audio-unmuted = 音频已取消静音。
+
+## PlayerProfileLogic
+label-loading-player-profile = 正在加载玩家档案……
+label-loading-player-profile-failed = 玩家档案加载失败。
+
+## ProductionTooltipLogic, EncyclopediaLogic
+label-requires = 需要{ $prerequisites }。
+
+## ReplayBrowserLogic
+label-duration = 时长：{ $time }
+
+options-replay-type =
+    .singleplayer = 单人游戏
+    .multiplayer = 多人游戏
+
+options-winstate =
+    .victory = 胜利
+    .defeat = 失败
+
+options-replay-date =
+    .today = 今天
+    .last-week = 最近 7 天
+    .last-fortnight = 最近 14 天
+    .last-month = 最近 30 天
+
+options-replay-duration =
+    .very-short = 5 分钟以内
+    .short = 短（10 分钟）
+    .medium = 中（30 分钟）
+    .long = 长（60 分钟以上）
+
+dialog-rename-replay =
+    .title = 重命名回放
+    .prompt = 输入新文件名：
+    .confirm = 重命名
+
+dialog-delete-replay =
+    .title = 删除所选回放？
+    .prompt = 删除回放 { $replay }？
+    .confirm = 删除
+
+dialog-delete-all-replays =
+    .title = 删除全部所选回放？
+    .prompt =
+    { $count ->
+       *[other] 删除 { $count } 个回放。
+    }
+    .confirm = 全部删除
+
+notification-replay-deletion-failed = 删除回放文件「{ $file }」失败，详见 debug.log。
+
+## ReplayUtils
+-incompatible-replay-recorded = 录制时使用的是
+
+dialog-incompatible-replay =
+    .title = 回放不兼容
+    .prompt = 无法读取回放元数据。
+    .confirm = 确定
+    .prompt-unknown-version = { -incompatible-replay-recorded }未知版本。
+    .prompt-unknown-mod = { -incompatible-replay-recorded }未知模组。
+    .prompt-unavailable-mod = { -incompatible-replay-recorded }不可用的模组：{ $mod }。
+    .prompt-incompatible-version = { -incompatible-replay-recorded }不兼容的版本：
+    { $version }。
+    .prompt-unavailable-map = { -incompatible-replay-recorded }不可用的地图：
+    { $map }。
+
+# SelectUnitsByTypeHotkeyLogic
+nothing-selected = 未选中任何单位。
+
+## SelectUnitsByTypeHotkeyLogic, SelectAllUnitsHotkeyLogic
+selected-units-across-screen =
+    { $units ->
+       *[other] 已选中屏幕内 { $units } 个单位。
+    }
+
+selected-units-across-map =
+    { $units ->
+       *[other] 已选中全图 { $units } 个单位。
+    }
+
+## ServerCreationLogic
+label-internet-server-nat-A = 互联网服务器（UPnP/NAT-PMP
+label-internet-server-nat-B-enabled = 已启用
+label-internet-server-nat-B-not-supported = 不支持
+label-internet-server-nat-B-disabled = 已禁用
+label-internet-server-nat-C = ）：
+
+label-local-server = 本地服务器：
+
+dialog-server-creation-failed =
+    .prompt = 无法监听端口 { $port }。
+    .prompt-port-used = 请检查端口是否已被占用。
+    .prompt-error = 错误：「{ $message }」（{ $code }）。
+    .title = 服务器创建失败
+    .cancel = 返回
+
+## ServerListLogic
+label-players-online-count =
+    { $players ->
+       *[other] { $players } 名玩家在线
+    }
+
+label-search-status-failed = 查询服务器列表失败。
+label-search-status-no-games = 未找到游戏，请尝试更改筛选条件。
+label-no-server-selected = 未选择服务器
+
+label-map-status-searching = 搜索中……
+label-map-classification-unknown = 未知地图
+
+label-players-count =
+    { $players ->
+        [0] 无玩家
+        [one] 1 名玩家
+       *[other] { $players } 名玩家
+    }
+
+label-bots-count =
+    { $bots ->
+        [0] 无电脑
+        [one] 1 个电脑
+       *[other] { $bots } 个电脑
+    }
+
+## ServerListLogic, ReplayBrowserLogic, ObserverShroudSelectorLogic
+label-players = 玩家
+
+## ServerListLogic, GameInfoStatsLogic
+label-spectators = 观战者
+label-spectators-count =
+    { $spectators ->
+        [0] 无观战者
+        [one] 1 名观战者
+       *[other] { $spectators } 名观战者
+    }
+
+## ServerlistLogic, GameInfoStatsLogic, ObserverShroudSelectorLogic, SpawnSelectorTooltipLogic, ReplayBrowserLogic
+label-team-name = 队伍 { $team }
+label-no-team = 无队伍
+
+label-playing = 进行中
+label-waiting = 等待中
+
+label-other-players-count =
+    { $players ->
+        [one] 另 1 名玩家
+       *[other] 另 { $players } 名玩家
+    }
+
+label-in-progress-for =
+    { $minutes ->
+        [0] 已进行不到 1 分钟。
+       *[other] 已进行 { $minutes } 分钟。
+    }
+
+label-password-protected = 需要密码
+label-waiting-for-players = 等待玩家
+label-server-shutting-down = 服务器正在关闭
+label-unknown-server-state = 服务器状态未知
+
+## Game
+notification-saved-screenshot = 已保存截图 { $filename }
+
+## ChatCommands
+notification-invalid-command = { $name } 不是有效命令。
+
+## DebugVisualizationCommands
+description-combat-geometry = 切换战斗几何覆盖层。
+description-render-geometry = 切换渲染几何覆盖层。
+description-screen-map-overlay = 切换屏幕地图覆盖层。
+description-depth-buffer = 切换深度缓冲覆盖层。
+description-actor-tags-overlay = 切换单位标签覆盖层。
+
+## DevCommands
+notification-cheats-disabled = 作弊已禁用。
+notification-invalid-cash-amount = 无效的资金数量。
+description-toggle-visibility = 切换可见性检查与小地图。
+description-give-cash = 给予默认或指定数量的资金。
+description-give-cash-all = 给予所有玩家与 AI 默认或指定数量的资金。
+description-instant-building = 切换瞬间建造。
+description-build-anywhere = 切换任意地点建造。
+description-unlimited-power = 切换无限电力。
+description-enable-tech = 切换全科技建造。
+description-fast-charge = 切换支援技能快速充能。
+description-dev-cheat-all = 开启全部作弊并附赠资金。
+description-dev-crash = 使游戏崩溃。
+description-levelup-actor = 为选中单位增加指定等级。
+description-player-experience = 为选中单位的所有者增加指定玩家经验。
+description-power-outage = 使选中单位的所有者停电 5 秒。
+description-kill-selected-actors = 杀死选中单位。
+description-dispose-selected-actors = 移除选中单位。
+
+## HelpCommands
+notification-available-commands = 可用命令如下：
+description-no-description = 暂无描述。
+description-help-description = 提供各种命令的实用信息。
+
+## PlayerCommands
+description-pause-description = 暂停或恢复游戏。
+description-surrender-description = 自毁一切并输掉游戏。
+
+## DeveloperMode
+notification-cheat-used = 使用作弊：{ $player } 使用了 { $cheat }{ $suffix }。
+
+## CustomTerrainDebugOverlay
+description-custom-terrain-debug-overlay = 切换自定义地形调试覆盖层。
+
+## CellTriggerOverlay
+description-cell-triggers-overlay = 切换脚本触发器覆盖层。
+
+## ExitsDebugOverlay
+description-exits-overlay = 显示工厂出口。
+
+## HierarchicalPathFinderOverlay
+description-hpf-debug-overlay = 切换分层寻路覆盖层。
+
+## PathFinderOverlay
+description-path-debug-overlay = 切换寻路可视化。
+
+## TerrainGeometryOverlay
+description-terrain-geometry-overlay = 切换地形几何覆盖层。
+
+## MapOptions, MissionBrowserLogic
+options-game-speed =
+    .slowest = 最慢
+    .slower = 较慢
+    .normal = 普通
+    .fast = 快
+    .faster = 较快
+    .fastest = 最快
+
+## TimeLimitManager
+options-time-limit =
+    .no-limit = 无限制
+    .options =
+        { $minutes ->
+           *[other] { $minutes } 分钟
+        }
+
+notification-time-limit-expired = 时间限制已到。
+
+## EditorActorBrush
+notification-added-actor = 已添加 { $name }（{ $id }）
+
+## EditorCopyPasteBrush
+notification-copied-tiles =
+    { $amount ->
+       *[other] 已复制 { $amount } 个地块
+    }
+
+## EditorDefaultBrush
+notification-selected-area = 已选中区域 { $x },{ $y }（{ $width },{ $height }）
+notification-removed-area = 已移除区域 { $x },{ $y }（{ $width },{ $height }）
+notification-selected-actor = 已选中单位 { $id }
+notification-cleared-selection = 已清除选择
+notification-removed-actor = 已移除 { $name }（{ $id }）
+notification-removed-resource = 已移除 { $type }
+notification-moved-actor = 已将 { $id } 从 { $x1 },{ $y1 } 移动到 { $x2 },{ $y2 }
+
+## EditorResourceBrush
+notification-added-resource =
+    { $amount ->
+       *[other] 已添加 { $amount } 格 { $type }
+    }
+
+## EditorTileBrush
+notification-added-tile = 已添加地块 { $id }
+notification-filled-tile = 已填充地块 { $id }
+
+## EditorMarkerLayerBrush
+notification-added-marker-tiles =
+    { $amount ->
+       *[other] 已添加 { $amount } 个 { $type } 类型标记地块
+    }
+notification-removed-marker-tiles =
+    { $amount ->
+       *[other] 已移除 { $amount } 个标记地块
+    }
+notification-cleared-selected-marker-tiles = 已清除 { $amount } 个 { $type } 类型标记地块
+notification-cleared-all-marker-tiles = 已清除 { $amount } 个标记地块
+
+## EditorActionManager
+notification-opened = 已打开
+
+## MapOverlaysLogic
+mirror-mode =
+    .none = 无
+    .flip = 翻转
+    .rotate = 旋转
+
+## ActorEditLogic
+notification-edited-actor = 已编辑 { $name }（{ $id }）
+notification-edited-actor-id = 已编辑 { $name }（{ $old-id }-> { $new-id }）
+
+## ConquestVictoryConditions, StrategicVictoryConditions
+notification-player-is-victorious = { $player } 获得胜利。
+notification-player-is-defeated = { $player } 已被击败。
+
+## OrderManager
+notification-desync-compare-logs = 第 { $frame } 帧失去同步。
+    请与其他玩家对比 syncreport.log。
+
+## SupportPowerTimerWidget
+support-power-timer = { $player } 的{ $support-power }：{ $time }
+
+## WidgetUtils
+label-win-state-won = 胜
+label-win-state-lost = 负
+
+## Player
+enumerated-bot-name =
+    { $name } { $number ->
+       *[zero] {""}
+        [other] { $number }
+    }
diff -ruN a/mods/common/fluent/zh-CN/hotkeys.ftl b/mods/common/fluent/zh-CN/hotkeys.ftl
--- a/mods/common/fluent/zh-CN/hotkeys.ftl	1969-12-31 16:00:00
+++ b/mods/common/fluent/zh-CN/hotkeys.ftl	2026-07-30 21:51:09
@@ -0,0 +1,186 @@
+hotkey-context =
+    .player = 玩家
+    .spectator = 观战者
+    .chat-input = 聊天输入
+    .menu = 菜单
+    .editor = 编辑器
+
+## chat.yaml
+hotkey-description-openteamchat = 打开队伍聊天
+hotkey-description-opengeneralchat = 打开全体聊天
+hotkey-description-togglechatmode = 切换聊天模式
+hotkey-description-autocomplete = 自动补全
+
+## control-groups.yaml
+hotkey-description-controlgroupselect01 = 选择编队 1
+hotkey-description-controlgroupselect02 = 选择编队 2
+hotkey-description-controlgroupselect03 = 选择编队 3
+hotkey-description-controlgroupselect04 = 选择编队 4
+hotkey-description-controlgroupselect05 = 选择编队 5
+hotkey-description-controlgroupselect06 = 选择编队 6
+hotkey-description-controlgroupselect07 = 选择编队 7
+hotkey-description-controlgroupselect08 = 选择编队 8
+hotkey-description-controlgroupselect09 = 选择编队 9
+hotkey-description-controlgroupselect10 = 选择编队 0
+hotkey-description-controlgroupcreate01 = 创建编队 1
+hotkey-description-controlgroupcreate02 = 创建编队 2
+hotkey-description-controlgroupcreate03 = 创建编队 3
+hotkey-description-controlgroupcreate04 = 创建编队 4
+hotkey-description-controlgroupcreate05 = 创建编队 5
+hotkey-description-controlgroupcreate06 = 创建编队 6
+hotkey-description-controlgroupcreate07 = 创建编队 7
+hotkey-description-controlgroupcreate08 = 创建编队 8
+hotkey-description-controlgroupcreate09 = 创建编队 9
+hotkey-description-controlgroupcreate10 = 创建编队 0
+hotkey-description-controlgroupaddto01 = 加入编队 1
+hotkey-description-controlgroupaddto02 = 加入编队 2
+hotkey-description-controlgroupaddto03 = 加入编队 3
+hotkey-description-controlgroupaddto04 = 加入编队 4
+hotkey-description-controlgroupaddto05 = 加入编队 5
+hotkey-description-controlgroupaddto06 = 加入编队 6
+hotkey-description-controlgroupaddto07 = 加入编队 7
+hotkey-description-controlgroupaddto08 = 加入编队 8
+hotkey-description-controlgroupaddto09 = 加入编队 9
+hotkey-description-controlgroupaddto10 = 加入编队 0
+hotkey-description-controlgroupcombinewith01 = 与编队 1 合并
+hotkey-description-controlgroupcombinewith02 = 与编队 2 合并
+hotkey-description-controlgroupcombinewith03 = 与编队 3 合并
+hotkey-description-controlgroupcombinewith04 = 与编队 4 合并
+hotkey-description-controlgroupcombinewith05 = 与编队 5 合并
+hotkey-description-controlgroupcombinewith06 = 与编队 6 合并
+hotkey-description-controlgroupcombinewith07 = 与编队 7 合并
+hotkey-description-controlgroupcombinewith08 = 与编队 8 合并
+hotkey-description-controlgroupcombinewith09 = 与编队 9 合并
+hotkey-description-controlgroupcombinewith10 = 与编队 0 合并
+hotkey-description-controlgroupjumpto01 = 跳转至编队 1
+hotkey-description-controlgroupjumpto02 = 跳转至编队 2
+hotkey-description-controlgroupjumpto03 = 跳转至编队 3
+hotkey-description-controlgroupjumpto04 = 跳转至编队 4
+hotkey-description-controlgroupjumpto05 = 跳转至编队 5
+hotkey-description-controlgroupjumpto06 = 跳转至编队 6
+hotkey-description-controlgroupjumpto07 = 跳转至编队 7
+hotkey-description-controlgroupjumpto08 = 跳转至编队 8
+hotkey-description-controlgroupjumpto09 = 跳转至编队 9
+hotkey-description-controlgroupjumpto10 = 跳转至编队 0
+hotkey-description-removefromcontrolgroup = 移出编队
+
+## editor.yaml
+hotkey-description-editorundo = 撤销
+hotkey-description-editorredo = 重做
+hotkey-description-editorcopy = 复制
+hotkey-description-editorquicksave = 保存地图
+hotkey-description-editorpaste = 粘贴
+hotkey-description-editordeleteselection = 删除所选
+hotkey-description-editorselecttab = 选择选项卡
+hotkey-description-editortilestab = 地块选项卡
+hotkey-description-editoroverlaystab = 覆盖物选项卡
+hotkey-description-editoractorstab = 单位选项卡
+hotkey-description-editortoolstab = 工具选项卡
+hotkey-description-editorhistorytab = 历史选项卡
+hotkey-description-editorsettingstab = 设置选项卡
+hotkey-description-editortogglegridoverlay = 网格覆盖层
+hotkey-description-editortogglebuildableoverlay = 可建造地形覆盖层
+hotkey-description-editortogglemarkeroverlay = 标记层覆盖层
+
+## game.yaml
+hotkey-description-cyclebase = 跳转至基地
+hotkey-description-tolastevent = 跳转至最近的雷达事件
+hotkey-description-toselection = 跳转至所选目标
+hotkey-description-selectallunits = 选择全部作战单位
+hotkey-description-selectunitsbytype = 按类型选择单位
+hotkey-description-cycleharvesters = 循环选择采矿车
+hotkey-description-pause = 暂停 / 恢复
+hotkey-description-sell = 出售模式
+hotkey-description-repair = 维修模式
+hotkey-description-placebeacon = 放置信标
+hotkey-description-cyclestatusbars = 切换状态栏显示
+hotkey-description-togglemute = 切换静音
+hotkey-description-toggleplayerstancecolor = 切换玩家关系颜色
+hotkey-description-takescreenshot = 截图
+hotkey-description-attackmove = 攻击移动
+hotkey-description-stop = 停止
+hotkey-description-scatter = 散开
+hotkey-description-deploy = 部署
+hotkey-description-guard = 警戒
+hotkey-description-stanceattackanything = 主动攻击
+hotkey-description-stancedefend = 防御
+hotkey-description-stancereturnfire = 还击
+hotkey-description-stanceholdfire = 停火
+hotkey-description-stopmusic = 停止
+hotkey-description-pausemusic = 暂停或继续
+hotkey-description-prevmusic = 上一首
+hotkey-description-nextmusic = 下一首
+
+## observer.yaml
+hotkey-description-observercombinedview = 所有玩家
+hotkey-description-observerworldview = 关闭迷雾
+hotkey-description-replayspeedslow = 慢速
+hotkey-description-replayspeedregular = 常速
+hotkey-description-replayspeedfast = 快速
+hotkey-description-replayspeedmax = 最高速
+hotkey-description-statisticsnone = 关闭统计
+hotkey-description-statisticsbasic = 基本统计
+hotkey-description-statisticseconomy = 经济统计
+hotkey-description-statisticsproduction = 生产统计
+hotkey-description-statisticssupportpowers = 支援技能统计
+hotkey-description-statisticscombat = 战斗统计
+hotkey-description-statisticsarmy = 军队统计
+hotkey-description-statisticsgraph = 统计图表
+hotkey-description-statisticsarmygraph = 军力图表
+
+## production-common.yaml
+hotkey-description-cycleproductionbuildings = 下一个生产设施
+hotkey-description-selectproductionbuilding = 当前生产设施
+hotkey-description-production01 = 栏位 01
+hotkey-description-production02 = 栏位 02
+hotkey-description-production03 = 栏位 03
+hotkey-description-production04 = 栏位 04
+hotkey-description-production05 = 栏位 05
+hotkey-description-production06 = 栏位 06
+hotkey-description-production07 = 栏位 07
+hotkey-description-production08 = 栏位 08
+hotkey-description-production09 = 栏位 09
+hotkey-description-production10 = 栏位 10
+hotkey-description-production11 = 栏位 11
+hotkey-description-production12 = 栏位 12
+hotkey-description-production13 = 栏位 13
+hotkey-description-production14 = 栏位 14
+hotkey-description-production15 = 栏位 15
+hotkey-description-production16 = 栏位 16
+hotkey-description-production17 = 栏位 17
+hotkey-description-production18 = 栏位 18
+hotkey-description-production19 = 栏位 19
+hotkey-description-production20 = 栏位 20
+hotkey-description-production21 = 栏位 21
+hotkey-description-production22 = 栏位 22
+hotkey-description-production23 = 栏位 23
+hotkey-description-production24 = 栏位 24
+
+## supportpowers.yaml
+hotkey-description-supportpower01 = 栏位 01
+hotkey-description-supportpower02 = 栏位 02
+hotkey-description-supportpower03 = 栏位 03
+hotkey-description-supportpower04 = 栏位 04
+hotkey-description-supportpower05 = 栏位 05
+hotkey-description-supportpower06 = 栏位 06
+
+## viewport.yaml
+hotkey-description-mapscrollup = 向上滚动
+hotkey-description-mapscrolldown = 向下滚动
+hotkey-description-mapscrollleft = 向左滚动
+hotkey-description-mapscrollright = 向右滚动
+hotkey-description-mapjumptotopedge = 跳转至上边缘
+hotkey-description-mapjumptobottomedge = 跳转至下边缘
+hotkey-description-mapjumptoleftedge = 跳转至左边缘
+hotkey-description-mapjumptorightedge = 跳转至右边缘
+hotkey-description-mapbookmarksave01 = 记录书签 1
+hotkey-description-mapbookmarkrestore01 = 跳转至书签 1
+hotkey-description-mapbookmarksave02 = 记录书签 2
+hotkey-description-mapbookmarkrestore02 = 跳转至书签 2
+hotkey-description-mapbookmarksave03 = 记录书签 3
+hotkey-description-mapbookmarkrestore03 = 跳转至书签 3
+hotkey-description-mapbookmarksave04 = 记录书签 4
+hotkey-description-mapbookmarkrestore04 = 跳转至书签 4
+hotkey-description-zoomin = 放大
+hotkey-description-zoomout = 缩小
+hotkey-description-resetzoom = 重置缩放
diff -ruN a/mods/common/fluent/zh-CN/rules.ftl b/mods/common/fluent/zh-CN/rules.ftl
--- a/mods/common/fluent/zh-CN/rules.ftl	1969-12-31 16:00:00
+++ b/mods/common/fluent/zh-CN/rules.ftl	2026-07-30 21:51:09
@@ -0,0 +1,61 @@
+## Shroud
+checkbox-fog-of-war =
+    .label = 战争迷雾
+    .description = 需要视野才能看到敌军
+
+checkbox-explored-map =
+    .label = 已探索地图
+    .description = 初始即揭开地图迷雾
+
+## DeveloperMode
+checkbox-debug-menu =
+    .label = 调试菜单
+    .description = 启用作弊与开发者命令
+
+## CrateSpawner
+checkbox-crates =
+    .label = 补给箱
+    .description = 用单位拾取补给箱以获得随机奖励或惩罚
+
+## MapBuildRadius
+checkbox-ally-build-radius =
+    .label = 依托盟友建造
+    .description = 允许盟友在你的建造范围内放置建筑
+
+checkbox-build-radius =
+    .label = 限制建造范围
+    .description = 建筑只能放置在建造厂周围范围内
+
+## MapOptions
+checkbox-short-game =
+    .label = 快速游戏
+    .description = 基地被摧毁即战败
+
+dropdown-tech-level =
+    .label = 科技等级
+    .description = 玩家可使用的单位与能力
+
+## MapOptions
+dropdown-game-speed =
+    .label = 游戏速度
+    .description = 控制时间流逝的速率
+
+## MapStartingLocations
+checkbox-separate-team-spawns =
+    .label = 队伍分散出生
+    .description = 未指定出生点的玩家将尽可能远离敌方玩家出生
+
+## SpawnStartingUnits
+dropdown-starting-units =
+    .label = 初始单位
+    .description = 游戏开始时玩家拥有的单位
+
+## TimeLimitManager
+dropdown-time-limit =
+    .label = 时间限制
+    .description = 时间结束时得分最高的玩家或队伍获胜
+
+## Tooltip
+label-tooltip-prefix =
+    .ally = 盟友
+    .enemy = 敌人
